src/tags.json

```
@@ -103,6 +103,7 @@
   "paperclip": ["attachment"],
   "pause": ["music", "stop"],
   "pause-circle": ["music", "stop"],
+  "pen-tool": ["pen", "vector", "drawing"],
```

"pen" doesn't need to be in the list because it's in the name 👍 

```suggestion
  "pen-tool": ["vector", "drawing"],
```